pkg/audit/manager.go

```diff
@@ -335,6 +347,14 @@ func (am *Manager) auditResources(
 	for gv, gvKinds := range clusterAPIResources {
 	kindsLoop:
 		for kind := range gvKinds {
+			// delete all existing folders from cache dir before starting next kind
+			err := am.removeAllFromDir(*apiCacheDir, int(*auditChunkSize))
```

Comment:
Why did this move into the loop? Are we trying to reduce peak disk usage?

Reply:
I think so?

Could be useful if there are a lot of large lists of kinds.